packages/vscode-extension/src/panels/LaunchConfigController.ts

```diff
@@ -17,6 +18,42 @@ import { getLaunchConfiguration } from "../utilities/launchConfiguration";
 
 const CUSTOM_APPLICATION_ROOTS_KEY = "custom_application_roots_key";
 
+function readApplicationRoot(appRootPath: string): ApplicationRoot {
+  const appRootAbsolutePath = path.resolve(workspace.workspaceFolders![0].uri.fsPath, appRootPath);
+  try {
+    const appRootConfig = require(appRootAbsolutePath + "/app.json");
```

Comment:
assuming we just want to read a json file here we should use requireNoCache here

Reply:
For sure :+1:
I read it using `fs` initially, and later found some place where we used `require` for it, so I just copied the approach here.